docs/journalist/sources.rst

```diff
@@ -3,17 +3,54 @@ Communicating with sources
 
 .. include:: ../includes/top-warning.rst
 
+SecureDrop Workstation lets journalists check SecureDrop, decrypt and securely
+view submissions, and reply to sources, all on the same computer.
+
+Once logged in, you will see a chat-interface-like screen, showing all of your
+sources in the middle panel. Each source is identified to you with a two word
+pseudonym.
+
 Opening a conversation
 ----------------------
 
+Clicking on a source will display their messages to you in the righthand panel.
+Replies that you or anyone from your organization has sent will also appear in
+the conversation flow.
+
+.. note:: If you are prompted by a dialog that says “Do you allow VM ‘sd-app’
+   to access your GPG keys (now and for the following 28800 seconds)?”, click
+   **Yes**.
```

Comment:
Let's give a bit of context here why we're asking them to say "Yes", to avoid training them to say "Yes" reflexively.

Reply:
See 1447bdd - how do we feel about this explanation?

Reply:
It doesn't need the bit about `sd-app` as qubes-rpc rules restrict access to sd-gpg, also feels a bit uneven in terms of level of technical detail. Otherwise good.

Reply:
fixed in 203b8d8